### PR TITLE
Bug 2028160:  Remove i18nKey in network-policy-peer-selectors.tsx

### PR DESCRIPTION
**Pull request by TimG1997**
@rebeccaalpert Make sure that the translation of the affected pods link works correctly and appears in correct form in the console-app.json file.

Fixes https://bugzilla.redhat.com/show_bug.cgi?id=2028160

### CODE REVIEW

File: frontend/packages/console-app/src/components/network-policies/network-policy-peer-selectors.tsx [hunk 1]
```diff
@@ -63,18 +60,33 @@ export const NetworkPolicyPeerSelectors: React.FC<PeerSelectorProps> = (props) =
...
-          >
-            affected pods
-          </Button>{' '}
-          that this <RuleName /> will apply to
-        </Trans>
+        {props.direction === 'ingress' ? (
+          <Trans ns="console-app">
+            Show a preview of the{' '}
+            <Button
+              data-test={`show-affected-pods-${props.direction}`}
```
Comment (mariomac) [inline frontend/packages/console-app/src/components/network-policies/network-policy-peer-selectors.tsx:67]: minor comment: maybe then you can replace this line by:
```suggestion
              data-test={`show-affected-pods-ingress`}
```

File: frontend/packages/console-app/src/components/network-policies/network-policy-peer-selectors.tsx [hunk 2]
```diff
@@ -63,18 +60,33 @@ export const NetworkPolicyPeerSelectors: React.FC<PeerSelectorProps> = (props) =
...
+            >
+              affected pods
+            </Button>{' '}
+            that this ingress rule will apply to.
+          </Trans>
+        ) : (
+          <Trans ns="console-app">
+            Show a preview of the{' '}
+            <Button
+              data-test={`show-affected-pods-${props.direction}`}
```
Comment (mariomac) [inline frontend/packages/console-app/src/components/network-policies/network-policy-peer-selectors.tsx:80]: minor comment: maybe you can replace this line by:
```suggestion
              data-test={`show-affected-pods-egress`}
```

**Review by mariomac — APPROVED** [review]

**Review by rebeccaalpert — CHANGES_REQUESTED** [review]
This looks good to me. I'm not seeing an errors.

I'm happy to lgtm this once @mariomac's comments are resolved. (I they're making a good point.)

File: frontend/packages/console-app/src/components/network-policies/network-policy-peer-selectors.tsx (hunk 2 above)
Comment (rebeccaalpert) [reply frontend/packages/console-app/src/components/network-policies/network-policy-peer-selectors.tsx:80]: I would drop the `s here and move to a string. 

```suggestion
              data-test="show-affected-pods-egress"
```

File: frontend/packages/console-app/src/components/network-policies/network-policy-peer-selectors.tsx (hunk 1 above)
Comment (rebeccaalpert) [reply frontend/packages/console-app/src/components/network-policies/network-policy-peer-selectors.tsx:67]: I would drop the `s here and move to a string, but I think this is a good point.

```suggestion
              data-test="show-affected-pods-ingress"
```

**Review by rebeccaalpert — APPROVED** [review]
Looks good to me. Thanks!

Please squash your commits and reach out to an approver for final approval. (They are listed in the OWNERS file.)

**Review by rebeccaalpert — APPROVED** [review]
Looks good to me after the squash!

**Comment by jotak** [comment]
/approve
thanks @TimG1997 !

**Pull request merged**